src/main/flight/gps_rescue.h

```diff
@@ -39,6 +39,7 @@ typedef struct gpsRescue_s {
     uint8_t useMag;
     uint16_t targetLandingAltitudeM; //meters
     uint16_t targetLandingDistanceM; //meters
+    uint8_t allowClimbToMaxReachedAlt;
```

Comment:
I think just `climbToMaxReachedAlt` is easier to understand here.

Reply:
Ok